theia-cpp-docker/README.md

```diff
@@ -44,3 +44,4 @@ docker build --no-cache --build-arg version=latest --build-arg -t theia-cpp:late
 
 #### Build Options:
   - `--build-arg strip=true` strips the application to save space but with reduced debuggability.
+  - `--build-arg LLVM=clang` version you prefer, please set LLVM=9 to build on ppc64le platform.
```

Comment:
I think it may not be clear what value is expected for `--build-arg LLVM=`. Maybe instead use something like:
`--build-arg LLVM=<version>` version of LLVM tools (clangd, clang-tidy) to install.  Please set LLVM=9 to build on ppc64le platform

Reply:
@marcdumais-work README file updated as suggested.